test/suites/init.sh

```diff
@@ -12,7 +12,7 @@ test_lxd_autoinit() {
   kill_lxd "${LXD_INIT_DIR}"
 
   # lxd init --auto --storage-backend zfs
-  if [ "${LXD_BACKEND}" = "zfs" ]; then
+  if [ "${LXD_BACKEND}" = "zfs" ] && command -v zfs >/dev/null 2>&1; then
```

Comment:
Ah, that's what I asked before if that if `LXD_BACKEND` is enough. I thought it was for Jenkins but not for some of the Ubuntu tests...